drivers/rf-transceiver/talise/README.rst

```diff
@@ -0,0 +1,372 @@
+and auxiliary DAC/ADC operations. Key routines include:
+
+- ``TALISE_setGpioMonitorOut`` and ``TALISE_getGpioMonitorOut`` – Route
+   internal signals for external debugging.
+
+- ``TALISE_setGpIntMask``, ``TALISE_getGpIntMask``,
+   ``TALISE_getGpIntStatus``, and ``TALISE_gpIntHandler`` – Manage
+   general purpose interrupts.
+
+- ``TALISE_setupAuxDacs`` and ``TALISE_writeAuxDac`` – Configure and
+   update auxiliary DAC outputs.
+
+- ``TALISE_setAuxAdcPinModeGpio``, ``TALISE_getAuxAdcPinModeGpio``,
+   ``TALISE_startAuxAdc``, and ``TALISE_readAuxAdc`` – Set up and read
+   from the auxiliary ADC.
+
+- ``TALISE_setSpi2Enable`` and ``TALISE_getSpi2Enable`` – Control the
+   secondary SPI interface used for Tx attenuation.
+
+- ``TALISE_getTemperature`` – Retrieve on‑chip temperature sensor data
+   via the ARM processor, with ``talGetGpioErrorMessage`` providing
+   human‑readable error descriptions.
+
+Radio Control and Pin Management
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Radio control functions integrate firmware streaming, power management,
+and frequency configuration to enable dynamic adaptation of the
+transceiver. These routines handle control pin settings, LO
+configuration, and mapping between transmit and observation channels.
+Featured functions include:
+
+- ``TALISE_loadStreamFromBinary`` – Load the stream processor’s firmware
+   image.
+
+- ``TALISE_setArmGpioPins`` – Configure ARM‑assigned GPIOs for
+   time‑division duplex and radio control purposes.
+
+- ``TALISE_setRadioCtlPinMode`` and ``TALISE_getRadioCtlPinMode`` – Set
+   and query radio control pin modes.
+
+- ``TALISE_setOrxLoCfg`` and ``TALISE_getOrxLoCfg`` – Configure
+   observation receiver LO settings.
+
+- ``TALISE_radioOn``, ``TALISE_radioOff``, and ``TALISE_getRadioState``
+   – Control and monitor the overall radio power state.
+
+- ``TALISE_setRxTxEnable`` and ``TALISE_getRxTxEnable`` – Manage signal
+   path enablement and channel mapping.
+
+- ``TALISE_setTxToOrxMapping`` – Map Tx channels to ORx channels for
+    calibration purposes.
+  
+    Additional routines control RF PLL frequency, set loop filters,
+    manage frequency hopping (e.g., ``TALISE_setFhmConfig``,
+    ``TALISE_setFhmMode``, ``TALISE_setFhmHop``, etc.), and adjust
+    external LO outputs.
+
+Driver Initialization Example
+-----------------------------
+
+.. code-block:: c
+
+  #include <stdint.h>
+  #include <stdbool.h>
+  #include <stdio.h>
+  #include <stdlib.h>
+  #include <string.h>
+  #include "no_os_print_log.h"
+  #include "no_os_error.h"
+  #include "no_os_alloc.h"
+  #include "no_os_delay.h"
+  #include "no_os_util.h"
+  #include "no_os_spi.h"
+  #include "no_os_clk.h"
+  #include "adrv9009.h"
+
+    struct adrv9009_init_param init_param = {
+    .spi_init = &(struct no_os_spi_init_param){
+       .device_id		= 0,
+       .max_speed_hz	= 25000000,
+       .chip_select	= 1,
+       .mode			= NO_OS_SPI_MODE_0,
```

Comment:
these are not properly aligned, either use tabs and properly align them or use single whitespace and dont try to align